src/go/intelligence/main.go

```diff
@@ -36,12 +39,55 @@ func run() {
 	}
 }
 
+func runNoOfTimes() {
+	utils.NoOfAlertsBeforeIntModule = 0
```

Comment:
all of these counters are not atomic and therefore can be screwed in concurrent calls

Reply:
@vkuznet I have changed the counters into one data structure and they are being incremented/decremented in the pipeline execution only. I think concurrent calls issue can happen only in the pipeline. But, here we are making changes in the infinite loop and after each loop now I am resetting the counter data structure.